buildkite/run_check_deploy_docker_build.sh

```diff
@@ -4,20 +4,10 @@
 # built successfully. This check is done in the buildkite-assets repo to prevent
 # an unbuildable Dockerfile from being deployed to buildkite instances.
 
-DOCKER_IMAGE_NAME=check-heroku-deploy-image
+echo "--- Building and testing with panorama_command.sh"
+HEROKU_DEPLOY_PATH=deploy/heroku-deploy \
```

Comment:
This tells `panorama_command.sh` to build the local, updated `Dockerfile` instead of the bootstrapped version in `/usr/local/bin/heroku-deploy`.